groups/admin.py

```diff
@@ -1,20 +1,10 @@
 from django.contrib import admin
 
-from .models import Discussion, Group
 
-
-@admin.register(Group)
 class GroupAdmin(admin.ModelAdmin):
     filter_horizontal = ('moderators', 'members_if_private')
     exclude = ('watchers',)
 
-    class Meta:
-        model = Group
-
 
-@admin.register(Discussion)
 class DiscussionAdmin(admin.ModelAdmin):
     exclude = ('subscribers', 'ignorers')
```

Comment:
Can we please have a comment in here to explain that the classes are registered in the appconfig?